helpers/components/addInArticleAds.js

```diff
@@ -173,6 +174,11 @@ function insertAd(newContent, options) {
     newContent.push(options.inArticleDesktop300x250);
   }
 
+  // add feature article banner ads
+  if (options.inArticleDesktopBanner && Object.keys(options.inArticleDesktopBanner).length) {
```

Comment:
we're using `.length` because the `inArticleDesktopbanner` is an an object in and of itself. If there is no ad instance, that object will be empty.  We need to check to make sure that the ad object actually has a property, hence the `.length`